main.go

```diff
@@ -39,6 +42,27 @@ func init() {
 	log.SetOutput(os.Stdout)
 	log.SetLevel(log.DebugLevel)
 	log.SetReportCaller(false)
+	// Initialize OpenTelemetry Tracer
+	if _, err := initTracer(); err != nil {
+		log.Fatalf("failed to init tracer: %v", err)
+	}
+}
+
+// Setup tracing
+func initTracer() (*sdktrace.TracerProvider, error) {
```

Comment:
This change will add traceparent https request method to all api calls going to our SM server.

2025-07-21 13:46:54,501 [https-openssl-nio-8443-exec-5] [trace-id: f04b8e47e76657ad58d2daf27e11f4fe-b28c50656f3c9450]  (JerseyLoggingFilter.java:65) INFO  JerseyLoggingFilter [] - 277 * Server has received a request on thread https-openssl-nio-8443-exec-5
277 > GET https://10.200.105.131/mgmt/v1.2/rest/shares/%2Ftest-csi-share-2
277 > accept: application/json
277 > content-type: application/json
277 > cookie: JSESSIONID=C7E742325340DE13BC8C5A50068CC54E
277 > host: 10.200.105.131
277 > traceparent: 00-f04b8e47e76657ad58d2daf27e11f4fe-b28c50656f3c9450-01
277 > user-agent: Go-http-client/1.1
277 > x-user-agent: Hammerspace CSI Plugin/v1.2.8